Dshop/apps/products_catalogue/models.py

```diff
@@ -28,13 +29,18 @@ class Meta:
         verbose_name_plural = 'Categories'
 
 
-class Product(models.Model):
-    name = models.CharField(max_length=255)
+class Product(CatalogueItemModel):
     category = models.ForeignKey(Category, on_delete=models.CASCADE)
     price = models.DecimalField(max_digits=10, decimal_places=2)
     price_last_30_days = models.DecimalField(max_digits=10, decimal_places=2, null=True, blank=True)
```

Comment:
Please change name to `lowest_price_last_30_days`

I am also thinking we could add `lowest_price_last_30_days_changed_at` - then we can use this in templates and also use automation to change it on save. 
Actually, you could try to add logic for that using TDD. I can help or we can work on that on our next meeting on Thu.

Reply:
We changed approach.